tests/testcases/test_admin_request.py

```diff
@@ -40,6 +44,15 @@ def test_get_admin_change_page(self):
         expected = '/admin/tests/user/{}/'.format(user.pk)
         self.client.get.assert_called_with(expected)
 
+    @skipUnless(is_django_gte_19(), "Django 1.9 has updated the admin url.")
+    def test_get_admin_change_page_django_gte_19(self):
```

Comment:
I'd prefer to use the nice name for the new test and rename the test of (now) legacy functionality.